src/components/yeti/ElevationProfile.vue

```diff
@@ -268,6 +488,17 @@ export default {
         .attr('class', 'area')
         .attr('d', this.area);
 
+      // build areas line (dashed)
+      let areasDataForLines = JSON.parse(JSON.stringify(this.areasData));
```

Comment:
we can use [struturedClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) instead